src/workerd/api/trace.h

```diff
@@ -71,11 +82,7 @@ class TraceItem final: public jsg::Object {
   kj::ArrayPtr<jsg::Ref<TraceException>> getExceptions();
   kj::ArrayPtr<jsg::Ref<TraceDiagnosticChannelEvent>> getDiagnosticChannelEvents();
   kj::Maybe<kj::StringPtr> getScriptName();
-  // TODO(someday): we expose this as jsg::Optional for now, since that ensures that the property will
```

Comment:
Having looked at the other fields in here I don't actually think using `jsg::Optional` (such that unpopulated fields manifest as `undefined` in JS) is actually any more inconsistent than using `jsg/kj::Maybe` so I don't think it's actually worth changing it in the future, especially since it will lead to a subtle change in behaviour of an existing field.